src/modules/ce/hooks/useProjectCeVisibility.tsx

```diff
@@ -32,13 +33,20 @@ export function useProjectCeVisibility(project?: ProjectAllFieldsFragment) {
     const showOutgoingReferrals =
       sendsDirectReferrals && canManageOutgoingReferrals;
 
+    // When to enable the 'Legacy Referrals' tab:
+    //   1) if the user has "canManageIncomingReferrals". This is a legacy perm that enables the ability to manage incoming legacy referrals.
+    //   2) if the user has "canManageOutgoingReferrals" and outgoing CE referrals are _not_ enabled. This permission is shared across legacy and CE.
+    //      This means that if the project had legacy outgoing referrals, but now CE Outgoing Direct Referrals are enabled, the Legacy
+    //      Referral tab will no longer be visible. (Unless the user has "canManageIncomingReferrals", in which case the legacy tab always shows.)
+    const showLegacyReferrals =
+      project.access.canManageIncomingReferrals ||
+      (!sendsDirectReferrals && canManageOutgoingReferrals);
```

Comment:
This is not pretty, but it's the best thing I could come up with because of the fact that "canManageOutgoingReferrals" is shared across both referral systems.

Reply:
Makes sense to me, thanks for running through all the scenarios.